rad/src/types/array.rs

```diff
@@ -31,6 +34,26 @@ impl RadonArray {
         self.inner_type
     }
 
+    pub fn inner_type_name(&self) -> String {
```

Comment:
Wow this is wild. There's no reason for calling `self.inner_type()` all the time.